### PR TITLE
Documentation Update for connection details over missing modules

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Added connection to be used in notes for modules missing info.
 Depnds-On: https://github.com/ansible/ansible-zuul-jobs/pull/1236

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

File: docs/cisco.ios.ios_banner_module.rst [hunk 1]
```diff
@@ -86,7 +86,7 @@ Parameters
                 <td>
                         <div><b>Deprecated</b></div>
                         <div>Starting with Ansible 2.5 we recommend using <code>connection: network_cli</code>.</div>
-                        <div>For more information please see the <a href='https://docs.ansible.com/ansible/latest/network/user_guide/platform_ios.html'>https://docs.ansible.com/ansible/latest/network/user_guide/platform_ios.html</a>.</div>
+                        <div>For more information please see the <a href='../network/user_guide/platform_ios.html'>IOS Platform Options guide</a>.</div>
```
Comment (NilashishC) [inline docs/cisco.ios.ios_banner_module.rst:89]: This won't work, since the modules live outside of ansible/ansible.

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Pull request merged**